File: app/src/main/java/ru/artempugachev/popularmovies/model/VideoResponse.java
package ru.artempugachev.popularmovies.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

import ru.artempugachev.popularmovies.model.Video;

/**
 * Data class for video response
 */

public class VideoResponse {
    @SerializedName("id")
    private String id;


    @SerializedName("results")
    private List<Video> results;

    public List<Video> getResults() {
        return results;
    }
}

File: app/src/main/java/ru/artempugachev/popularmovies/model/VideoResponse.kt
package ru.artempugachev.popularmovies.model

import com.google.gson.annotations.SerializedName

import ru.artempugachev.popularmovies.model.Video

/**
 * Data class for video response
 */

data class VideoResponse (
    @SerializedName("id")
    val id: String,

    @SerializedName("results")
    val results: List<Video>
)
